custom/icds_reports/tests/agg_tests/fixtures/child_tasks.csv

```diff
@@ -1,134 +1,134 @@
-task_id_120,2017-04-01,st2,,da43e61f-68ce-471b-a735-6c25fab8f61c,,2016-10-11,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,s7
-task_id_121,2017-04-01,st1,,892c3922-f4b8-4f3b-b677-47b4983e8851,,2015-01-17,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,s4
-task_id_122,2017-04-01,st1,,0125bcaf-bba4-4fdb-bbd2-8021cda471be,,2014-06-04,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,s4
-task_id_123,2017-04-01,st1,,f884e0e3-e2ad-43ec-a853-d95d72c04ffd,,2014-01-18,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,s4
-task_id_124,2017-04-01,st1,,ccd5242e-7de7-4a92-9799-884a1840d926,,2013-09-03,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,s4
-task_id_125,2017-04-01,st1,,77626e3b-1d14-4288-aae1-4def7e7bb721,,2014-09-29,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,s4
-task_id_126,2017-04-01,st2,,baeee6e2-2375-49c2-a430-b0788257cc36,,2014-08-29,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,s8
-task_id_127,2017-04-01,st2,,1c39cf9e-a547-4b8b-af30-8505f46caef9,,2013-10-05,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,s8
-task_id_128,2017-04-01,st2,,3a71ab8f-0c40-484c-969b-8b720064f657,,2014-08-26,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,s8
-task_id_129,2017-04-01,st2,,17841ccf-4db5-46db-bbba-59c03e92194c,,2013-09-08,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,s8
-task_id_130,2017-04-01,st1,,89176335-0c1f-46b4-9be0-ab1e918b7855,,2014-06-27,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,s1
-task_id_131,2017-04-01,st1,,fd1b1c9a-4e4a-4ab8-95cb-76459bcedabf,,2012-06-23,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,s1
-task_id_132,2017-04-01,st1,,a6a79563-3249-42d3-85de-a7ef3f17acc7,,2017-05-27,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,s3
-task_id_133,2017-04-01,st2,,77ead103-699a-4f85-884f-dc5b9671c79b,,2017-05-23,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,s6
+doc_id,inserted_at,state_id,awc_id,child_health_case_id,tasks_type,immun_one_year_date,due_list_date_1g_dpt_1,due_list_date_2g_dpt_2,due_list_date_3g_dpt_3,due_list_date_5g_dpt_booster,due_list_date_5g_dpt_booster1,due_list_date_7gdpt_booster_2,due_list_date_0g_hep_b_0,due_list_date_1g_hep_b_1,due_list_date_2g_hep_b_2,due_list_date_3g_hep_b_3,due_list_date_3g_ipv,due_list_date_4g_je_1,due_list_date_5g_je_2,due_list_date_5g_measles_booster,due_list_date_4g_measles,due_list_date_0g_opv_0,due_list_date_1g_opv_1,due_list_date_2g_opv_2,due_list_date_3g_opv_3,due_list_date_5g_opv_booster,due_list_date_1g_penta_1,due_list_date_2g_penta_2,due_list_date_3g_penta_3,due_list_date_1g_rv_1,due_list_date_2g_rv_2,due_list_date_3g_rv_3,due_list_date_4g_vit_a_1,due_list_date_5g_vit_a_2,due_list_date_6g_vit_a_3,due_list_date_6g_vit_a_4,due_list_date_6g_vit_a_5,due_list_date_6g_vit_a_6,due_list_date_6g_vit_a_7,due_list_date_6g_vit_a_8,due_list_date_7g_vit_a_9,supervisor_id,due_list_date_anc_1,due_list_date_anc_2,due_list_date_anc_3,due_list_date_anc_4,due_list_date_tt_1,due_list_date_tt_2,due_list_date_tt_booster,due_list_date_1g_bcg
+task_id_02,2017-03-01,st1,a44,0a04f052-f32b-4874-b38b-d21244f81516,child,,,2017-03-01,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,s4,2017-03-01,2017-03-01,2017-03-01,2017-03-01,2017-03-01,2017-03-01,2017-03-01,2017-03-01
```

Comment:
I dont see any of the test output data is updated of child_health table and mbt. Could you update the fixture such that this change can be verified.

Reply:
i updated child_health_monthly_2017-04-01_sorted.csv and child_health_monthly_2017-05-01_sorted.csv. Do i have to add more changes?